examples/torch/object_detection/layers/modules/ssd_head.py

```diff
@@ -90,6 +90,7 @@ def forward(self, features, image_tensor):
 
         with no_nncf_trace():
             priors = self.prior_box(features, image_tensor).to(loc.device)
+        priors = priors.detach()
```

Comment:
Without detach priors would be model outputs with requires_grad=True, but actually priors are not used in loss calculations directly so they are not included in backpropagation graph (that is coded inside MultiBoxLoss). So if model outputs shouldn't be used for backpropagation they should have requires_grad=False. 
`priors = priors.detach()` excludes priors from a backpropagation graph.
Also because KD logic requires differentiation of model outputs with requires_grad=True than we don't want model outputs that shouldn't be differentiated have requires_grad=True (that would cause an error).

Reply:
Yes we discussed it. for me it's clear, but not for other developers and users. I meant adding comment in the code 😄